addon/mixins/touch-action.js

```diff
@@ -1,16 +1,19 @@
 import { get, defineProperty, computed } from '@ember/object';
 import Mixin from '@ember/object/mixin';
 import { isHTMLSafe, htmlSafe } from '@ember/string';
+import config from 'ember-get-config';
+
+const defaults = config.EmberHammertime || {};
 
 const FocusableInputTypes = ['button', 'submit', 'text', 'file'];
 // Set this to `false` to not apply the styles automatically to elements with an `action`
-const TouchActionOnAction = true;
+const TouchActionOnAction = (typeof defaults.touchActionOnAction == 'undefined') ? true : defaults.touchActionOnAction;
 // Remove 'onclick' if you do not want the styles automatically applied to elements with an `onclick`
-const TouchActionAttributes = ['onclick'];
+const TouchActionAttributes = defaults.touchActionAttributes || ['onclick'];
 // Remove whichever element types you do not want automatically getting styles applied to them
-const TouchActionSelectors = ['button', 'input', 'a', 'textarea'];
+const TouchActionSelectors = defaults.touchActionSelectors || ['button', 'input', 'a', 'textarea'];
 // The actual style string that is applied to the elements. You can tweak this if you want something different.
-const TouchActionProperties = 'touch-action: manipulation; -ms-touch-action: manipulation; cursor: pointer;';
+const TouchActionProperties = defaults.touchActionProperties || 'touch-action: manipulation; -ms-touch-action: manipulation; cursor: pointer;';
```

Comment:
There is a corresponding AST transform that we would want to line up with this. Either remove it or make that use the same configuration passed in from the apps environment.js. Otherwise styles will be out of sync depending on whether the element is dynamic(rendered by glimmer vm) or static (rendered by browser as normal html tag). We want them to be the same in both cases.

Reply:
The app config is already used in that stuff, just not in the mixin, I think.

Reply:
yeah - i don't know if ember-get-config will pass through its options to the index.js file where we forward those options to the AST parser... or how to reconcile that.... but we shouldn't allow native link or button elements to have different css classes than ember components (like link-to) as that'll cause an inconsistent experience for devs (unless they're completely aware of touch-action css property, which is funky to say the least)

I'll check this as reviewed by me, but this issue in particular, I think we should ensure is right before merging...

Reply:
@eriktrom we're already consuming the app config in index.js here https://github.com/html-next/ember-hammertime/blob/master/index.js#L55.

Reply:
ahh - yes thank you. looks like this will work out then using ember-get-config for the mixin, and that code for the ast parser.. config set once in environment.js. nice

(double rainbow, it does exist!)